packages/apple-pay-integration/tsconfig.lib.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.tsx"
+  ]
+}
```

Comment:
I don't think we need `tsconfig.lib.json`, are we going to remove it in phase 2 with our own custom generator?

Reply:
Till we move to phase 2 we might need to keep this for now.